docs/howtos/custom_builder_go.md

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_label: "Deploy Gitea with a custom builder image"
+sidebar_position: 14
+title: ""
+---
+
+# Deploy Gitea with a custom builder image
+
+This How To will show you how to deploy Gitea with Epinio and a custom builder.
+
+[Gitea](https://github.com/go-gitea/gitea) is a self-hosted Git service. It's written in Go and Node.js.
+
+At the time of writing the Paketo Go buildpack doesn't support Node.js asset compilation (see [issue #671](https://github.com/paketo-buildpacks/go/issues/671)), so we need to create a custom builder.
+
+This builder will just check the needed Go and Node dependencies, and execute the `make` command to build both the frontend and the backend.
+
+The code used during this guide is available in the [example-builder-gitea](https://github.com/epinio/example-builder-gitea) repository.
+
+
+## Basic Concepts and prerequisites
+
+Before starting it's useful to know some basic concepts around Buildpacks, and have some tools already installed.
+
+You can find in the Buildpack documentation what [stacks](https://buildpacks.io/docs/concepts/components/stack/), [builders](https://buildpacks.io/docs/concepts/components/builder/), and [buildpacks](https://buildpacks.io/docs/concepts/components/buildpack/) are. You should also have [`pack`](https://buildpacks.io/docs/tools/pack/) installed, along with `docker` and `skopeo` (to manage OCI artifacts).
+
+
+## Create the stack
+
+Unfortunately we were not able to reuse the [Paketo Full Stack](https://github.com/paketo-buildpacks/bionic-full-stack) because of a missing dependency in the `run` image. To make it simpler we just [forked](https://github.com/epinio/bionic-full-stack) this stack and added `git` to the run packages (see [408e949](https://github.com/epinio/bionic-full-stack/commit/408e949558a437c99858ac7bb99a8b78e78935e8)).
+
+> Note:  
+> We had to fork and/or reuse the same stack ID in order to reuse some Paketo buildpacks, that are going to work only with these stacks (see https://github.com/paketo-buildpacks/node-engine/blob/main/buildpack.toml#L29).
+
+### Build the stack and publish the image
+
+To build the stack:
+
+```
+./stacks/bionic-full-stack/scripts/create.sh
```

Comment:
Should mention that we are in the main directory of the checkout of the `example-builder-repository` (Well, I am assuming here that this is the working directory.

Reply:
I've added a couple of commands that we were showing the cloning and `cd` into the example repository. I guess it should be enough!